### DISCUSSION TITLE
1 uuid column to populate from 2 different table

### DISCUSSION THREAD

**Discussion by hibangun** [mikro-orm/mikro-orm#3047] [Q&A]
This is my tables
`titles`
```
{
  id: 'xxxx',
  name: 'xxxx',
  ...
}
```

`events`
```
{
  id: 'xxxx',
  name: 'xxxx',
  ...
}
```

`contents`
```
{
  id: 'xxxx',
  content_id: 'xxxx', <- id of titles or events table
  content_type: 'TITLES' / 'EVENTS'
  ...
}
```

How do I populate titles/events table when query `contents` table?
If I have foreign key in contents table i can just simply
```
const data = await em.find(Contents, whereQueries, {
   populate: ['content_title', 'content_event']
    orderBy,
    limit: ITEMS_PER_PAGE,
    offset: (page - 1) * ITEMS_PER_PAGE,
  })
```

but in this case i dont have any titles/events foreign key in `contents` table

**Comment by B4nan** [comment]
This is not supported, you need to have FKs if you want to populate things, and everything needs to be defined as relations. Defining relations that does not target a FK is currently not possible, it's basically this issue #593. I do have some WIP to address this, so it might be supported in some future 5.x release.

Actually reading this again, you probably want polymorphic relations (one FK that can target different tables), that as also not supported, the issue is here: #706.
